business_requirement_crm/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Tecnativa - Victor M.M. Torres
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Business Requirement CRM',
+    'category': 'Business Requirements Management',
+    'summary': 'Convert Leads to Business Requirement',
+    'version': '11.0.1.1.0',
+    'website': 'https://www.elico-corp.com/',
+    "author": "Elico Corp, Odoo Community Association (OCA)",
```

Comment:
Change to Tecnativa

Reply:
@sergio-teruel Why change `author`?

Reply:
Because this has been done by us entirely from scratch

Reply:
@pedrobaeza Thanks For a quick response.

FYI here v8, we have https://github.com/OCA/business-requirement/pull/8 , That's why I am asking you

But Good, Go Ahead with this PR

Reply:
If you look to both source codes, it has nothing to do with 8.0 one, as it has been totally rethought from the new ground we have made, and that's why you are only seeing 1 commit.